packages/runtime/src/components/_internal/ImplWrapper.tsx

```diff
@@ -154,7 +155,17 @@ const _ImplWrapper = React.forwardRef<HTMLDivElement, ImplWrapperProps>((props,
     };
   }, [c.id, stateManager.store]);
 
-  const mergedProps = { ...evaledComponentProperties, ...propsFromTraits };
+  const mergedProps = useMemo(() => {
+    const mergedProps: Record<string, any> = { ...evaledComponentProperties, ...propsFromTraits };
```

Comment:
`mergedProps` is declared outside, use other name.

Reply:
This needs fix.

Reply:
Done.